backend/routes/api.php

```diff
@@ -25,6 +25,9 @@
 Route::post('login', [AuthController::class, 'login']);
 Route::apiResource('auth', AuthController::class)->only('store');
 
+Route::post('forgot.password', [AuthController::class, 'forgotPassword']);
```

Comment:
is there any specific reason as to why " . " (dot) is being used here?

this is not a resourceful route so i dont think   .   is suited to be used here for declaring the route